### DISCUSSION TITLE
第九模拟宇宙挂机开放了吗

### DISCUSSION THREAD

**Discussion by Mikasa-777** [LmeSzinc/StarRailCopilot#463] [Q&A] [answered]
想知道是大大们还在开发中，还是自己这边src更新出问题了，或者是bug了。因为看issue和discussion没人提这个问题，第九模拟宇宙出来后也过了好久了，记得第八模拟宇宙还是更新很快的，所以就来问一下啦。

**Comment by MomentDerek** [comment, marked as answer]
第九世界有3d地图，根据小地图判断的方式失效了，现在全网都没有解决方案
